client/oidc/src/main/java/io/quarkiverse/openapi/generator/oidc/providers/OAuth2AuthenticationProvider.java

```diff
@@ -30,14 +31,30 @@ public OAuth2AuthenticationProvider(String name,
         validateConfig();
     }
 
+    public OAuth2AuthenticationProvider(String name,
+            String openApiSpecId, OidcClientRequestFilterDelegate delegate, List<OperationAuthInfo> operations) {
+        this(name, openApiSpecId, delegate, operations, new ConfigCredentialsProvider());
```

Comment:
Sorry I merged without noticing this change. This is wrong, you need to get the credProvider from the CDI, or your custom code won't be injected.

Reply:
too late :) https://github.com/quarkiverse/quarkus-openapi-generator/pull/1176